### ISSUE TITLE
Trying to convert from Swagger 1.1 to 2.0

### ISSUE THREAD

**Issue by tylerjames** [LucyBot-Inc/api-spec-converter#122]
I've attempted this both in JS and from the command line but have encountered the same problem in both.

I use a command similar to:
`api-spec-converter https://api.gettyimages.com/swagger/api-docs --from=swagger_1 --to=swagger_2 > swagger.json`

Except I point it at the Swagger 1.1 API listing file which looks something like this:
```
    "swaggerVersion": "1.1",
    "apis": [
        {
            "path": "/Authentication",
            "description": "login, logout of the dashboard, ping the server, error codes"
        },
        {
            "path": "/User",
            "description": "create/update/get/delete/migrate a user, retrieve users public info, password reset"
        },
```
etc.

The actual definitions of the APIs are contained in separate files corresponding to the path name and in the same directory as the listing file.

**The problem**
It seems like the tool is trying to navigate to `Authentication` as if it is in a directory that is under my `api-docs.json` directory listing file.
`Fatal Error: ENOTDIR: not a directory, open 'api-docs/api-docs.json/Authentication`

It's trying to open the definitions file as if it is a directory and it's failing.

Has anyone run into this before? Is converting from Swagger 1.1 supported?

Thanks

**Comment by MikeRalphson** [comment]
Can reproduce this when there is no `basePath` set.

Not sure of the differences in resolution rules between Swagger 1.0, 1.1. and 1.2, but [this line](https://github.com/apigee-127/swagger-converter/blob/master/index.js#L65) in swagger-converter looks suspect; changing it to include version 1.1 fixes the problem.

Hosting the file via a local web-server and referencing it via URL may also help.

**Comment by IvanGoncharov** [comment]
@MikeRalphson Resolving subresources in Swagger 1.x is a total mess.
Back then I wrote this code in `swagger-converter` I used `swagger-js` as reference:
https://github.com/swagger-api/swagger-js/blob/2.x/lib/spec-converter.js#L535-L543
Since it was used in `swagger-ui` that mean an absolute majority of specs following this algorithm.
I also run `swagger-converter` on couple hundreds of Swagger 1.x specs and it manage to resolve all including all `1.1` specs.

So I would advise against altering behavior for `1.1` since it will fix only one spec and broken many others.

As a workaround, `api-spec-converter` can support option for overriding URLs to subresources.
But I'm not sure how to implement it best.
